frontend/javascripts/oxalis/view/right-menu/trees_tab_view.js

```diff
@@ -116,22 +117,31 @@ type State = {
 
 export async function importTracingFiles(files: Array<File>, createGroupForEachFile: boolean) {
   try {
-    const wrappedAddTreesAndGroupsAction = (trees, treeGroups, groupName) => {
+    const wrappedAddTreesAndGroupsAction = (trees, treeGroups, groupName, userBoundingBoxes) => {
+      const actions =
+        userBoundingBoxes && userBoundingBoxes.length > 0
```

Comment:
According to the type signature, `parseNml` always returns an array for `userBoundingBoxes`, so the existence check should be obsolete.

Reply:
This is not correct. I only implemented the import of `userBoundingBoxes` for nmls and not for protobuf tracings. Thus `userBoundingBoxes` can still be undefined.